packages/extension-chakra-store-locator/package.json

```diff
@@ -18,7 +18,8 @@
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
-    "@chakra-ui/react": "2.8.2",
+    "@chakra-ui/react": "3.22.0",
+    "@emotion/react": "^11.14.0",
```

Comment:
it is likely we will remove this package, but I need to update to be consistent for entire monorepo. Otherwise, lerna won't install

Reply:
Is this something you plan to address in this PR?